packaging/rpm/microshift.spec

```diff
@@ -394,6 +394,7 @@ if [ $1 -eq 1 ]; then
 	# if crio was already started, restart it so it will catch /etc/crio/crio.conf.d/10-microshift.conf
 	systemctl is-active --quiet crio && systemctl restart --quiet crio || true
```

Comment:
We need to add a similar check for crio-wipe here, so that it's restarted if already running.